NuGet/ChakraCore.Debugger.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>ChakraCore.Debugger</id>
```

Comment:
Is it worth keeping this id in line with the chakracore nuget package, something like `Microosft.ChakraCore.Debugger` to match `Microsoft.ChakraCore`?

Reply:
Done.